test/unit/tokenizer_test.rb

```diff
@@ -20,6 +23,18 @@ def test_tokenize_blocks
 
     assert_equal [' ', '{%comment%}', ' ', '{%endcomment%}', ' '], tokenize(' {%comment%} {%endcomment%} ')
     assert_equal ['  ', '{% comment %}', ' ', '{% endcomment %}', ' '], tokenize("  {% comment %} {% endcomment %} ")
+
+    # Doesn't strip whitespace
+    assert_equal [' ', '  comment  ', ' '], tokenize(' {%  comment  %} ', trimmed: true)
+  end
+
+  def test_tokenize_for_liquid_tag
+    source = "\nfunk\n\n  so | brother   \n"
+
+    assert_equal ["\nfunk\n\n  ", "so | brother   \n"], tokenize(source, true)
```

Comment:
I'll change this for the test, but not for the Tokenizer constructor, since the plan is to refactor it anyway and I'd rather not write the inefficient C kwarg handling code.